### PR TITLE
Doc - 2571 Update Error Tracking collector url

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [X] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*
Updated the error tracking collector url from https://app.harness.io/gratis/et-collector/ to  https://collector.et.harness.io/prod1

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/service-reliability-management/use-service-reliability-management/error-tracking-category/install-the-error-tracking-agent.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ This option lets you install the Error Tracking Agent as a standalone. Perform t
   For example:
 
 ```
-ENV ET_COLLECTOR_URL=https://app.harness.io/prod1/et-collector/  
+ENV ET_COLLECTOR_URL=https://collector.et.harness.io/prod1/et-collector/
```
Comment (hodaya-gamliel) [inline docs/service-reliability-management/use-service-reliability-management/error-tracking-category/install-the-error-tracking-agent.md:86]: should be https://collector.et.harness.io/prod1/ rather then https://collector.et.harness.io/prod1/et-collector/

**Comment by SudheendraKatte** [comment]
Hi @hodaya-gamliel , I have addressed the review comments. Please look into the topic again and let me know.

**Comment by SudheendraKatte** [comment]
Hi @hodaya-gamliel , updated all the ET Collector url instances.

**Review by hodaya-gamliel — APPROVED** [review]

**Review by aimurphy — APPROVED** [review]
changes to the URLs LGTM

**Pull request merged**